consul/ci/consul.rake

```diff
@@ -0,0 +1,64 @@
+require 'ci/common'
+
+def consul_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def consul_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/consul_#{consul_version}"
+end
+
+namespace :ci do
+  namespace :consul do |flavor|
+    task before_install: ['ci:common:before_install']
+
+    task install: ['ci:common:install'] do
+      use_venv = in_venv
+      install_requirements('consul/requirements.txt',
+                           "--cache-dir #{ENV['PIP_CACHE']}",
+                           "#{ENV['VOLATILE_DIR']}/ci.log", use_venv)
+      # sample docker usage
+      # sh %(docker create -p XXX:YYY --name consul source/consul:consul_version)
+      # sh %(docker start consul)
```

Comment:
This is just the skeleton that was generated. I could try to spin one up, it probably shouldn't be too hard.

You think I should?